.storybook/preview.tsx

```diff
@@ -8,7 +8,13 @@ export const parameters = {
   actions: { argTypesRegex: '^on[A-Z].*' },
 }
 
+const localStorageResetDecorator = (Story) => {
```

Comment:
tried adding this because there was some weird stuff happening on banner component page, but this didn't do it